superset/models/slice.py

```diff
@@ -380,9 +380,7 @@ def event_after_chart_changed(
     _mapper: Mapper, _connection: Connection, target: Slice
 ) -> None:
     cache_chart_thumbnail.delay(
-        current_user=get_current_user(),
-        chart_id=target.id,
-        force=True,
+        current_user=get_current_user(), chart_id=target.id, force=True
```

Comment:
Hmm, I'd maybe be ok with forcefully invalidating thumbs on chart/dashboard change, but recaching it? I'd propose adding a config flag `THUMBNAIL_ON_CHANGE: "invalidate" | "recalculate" | None` for admins to be able to specify if they want to just ignore changes, invalidate or recache every time the object changes.

Reply:
discussed on slack - out of scope